db/migrate/20160619064544_change_memory_reserve_data_type_in_vm.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMemoryReserveDataTypeInVm < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :memory_reserve, :bigint
```

Comment:
why does this need to be a bigint?

would the other memory related columns need that bump too?

have you looked into the other usages of `memory_reserve` in our DB?
I see that column in metrics (which is probably different from this) and in resource_pools. 
Not sure about how they relate